src/rocksdb_options.rs

```diff
@@ -77,8 +77,8 @@ impl BlockBasedOptions {
         }
     }
 
-    pub fn set_lru_cache(&mut self, size: size_t) {
-        let cache = crocksdb_ffi::new_cache(size);
+    pub fn set_lru_cache(&mut self, size: size_t, num_shard_bits: c_int, strict_capacity_limit: c_uchar, high_pri_pool_ratio: c_double) {
```

Comment:
Please add comment to demonstrate how to set `num_shard_bits` and `high_pri_pool_ratio`.

Reply:
fixed